roles/core/cluster/tasks/main.yml

```diff
@@ -15,3 +15,7 @@
 
 - import_tasks: finalize.yml
   tags: always
+
+# cluster node remover playbook
+- import_tasks: removenode.yml
+  tags: always
```

Comment:
Should this also have a tag for "removenode" or something... I'm wondering how we may want to have control so that we can only call these tasks under the role ....   

I don't know... just talking out loud.